quantecon/tests/test_kalman.py

```diff
@@ -53,7 +53,7 @@ def test_stationarity(self):
 
 
     def test_update_using_stationary(self):
-        A, Q, G, R = self.A, self.Q, self.G, self.R
+        # A, Q, G, R = self.A, self.Q, self.G, self.R  # unused
```

Comment:
@natashawatkins would you have time to check this?

Reply:
not used in this function - was probably just copied from above